.github/workflows/publish-dev.yaml

```diff
@@ -18,12 +18,14 @@ jobs:
         with:
           path: pages
           fetch-depth: 0
+          submodules: recursive
```

Comment:
Note that now we are effectively checking out the protobuf repo twice – once through a submodule, and once in lines 23–27.

Please file an issue in this GitHub tracker to simplify this. This is only a matter of updating the CI scripts.

Reply:
> Note that now we are effectively checking out the protobuf repo twice – once through a submodule, and once in lines 23–27.

> Please file an issue in this GitHub tracker to simplify this. This is only a matter of updating the CI scripts.

This is not done.